x/inflation/types/keys.go

```diff
@@ -46,7 +46,7 @@ var LastHalvenEpochKey = []byte{0x03}
 
 const (
 	// module acct name for developer vesting
-	DeveloperVestingModuleAcctName = "developer_vesting_unvested"
+	TeamVestingSupplyModuleAcctName = "team_vesting_supply"
```

Comment:
```suggestion
	TeamModuleAcctName = "team_account"
```

Reply:
also open to call this `tharsis_account`

Reply:
I like `tharsis_account`